Cargo.toml

```diff
@@ -130,3 +131,7 @@ is-it-maintained-open-issues = { repository = "paritytech/substrate" }
 [profile.release]
 # Substrate runtime requires unwinding.
 panic = "unwind"
+
+[patch.crates-io]
```

Comment:
Those lines can be removed after https://github.com/paritytech/parity-common/pull/250 is merged and `impl-rlp 0.2.1` is released.